assets/global.js

```diff
@@ -819,6 +820,41 @@ class VariantSelects extends HTMLElement {
     });
   }
 
+  updateVariantStatuses() {
+    const selectedOptionOneSelector = this.id.includes('variant-radios') ? '[type=radio]:checked+label' : 'option:checked';
+    const typeOfInput = this.id.includes('variant-radios') ? 'input' : 'option';
+
+    //Grab all the existing variants based on the selected option 1
+    this.selectedOptionOneVariants = JSON.parse(this.querySelector('[type="application/json"]').textContent).filter(variant => this.querySelector(selectedOptionOneSelector).innerText.trim() === variant.option1);
+    //Array of all the input wrappers we have (option1, option2 and option3)
```

Comment:
Are we always only going to have 3 options?

Reply:
For now yeah. It's a limitation Shopify has had for... ever I think? So we're limited to 3 options with a total limit of 100 variants per products ([here is more info](https://help.shopify.com/en/manual/products/variants/add-variants))